bentoml/_internal/frameworks/mlflow.py

```diff
@@ -4,7 +4,6 @@
 import shutil
 import typing as t
 import logging
-import tempfile
```

Comment:
this is still required, right?

Reply:
I imported this lazily

Reply:
probably no need to do that for built-in libraries